webrender/src/device/mod.rs

```diff
@@ -3,5 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 mod gl;
+pub mod query_gl;
```

Comment:
Without the `pub` we can't use `device::query` for `use` declarations, if it's OK to write e.g. `use device::GpuSampler` instead of `use device::query::GpuSampler`, I will change to that.

Reply:
Or we can write `pub(crate)` in line 6 and 9, if we want to keep those `device::query` paths.